### DISCUSSION TITLE
OSS项目推荐网站汇总

### DISCUSSION THREAD

**Discussion by xgdyp** [X-lab2017/open-wonderland#69] [1️⃣ 新鲜事]
我们都知道awesomeopensource是一个很不错的oss分类网站

下面这篇文章介绍了很多类似的网站

https://www.pixstacks.com/awesome-open-source

**Comment by will-ww** [comment]
> 我们都知道awesomeopensource是一个很不错的oss分类网站
> 
> 下面这篇文章介绍了很多类似的网站
> 
> https://www.pixstacks.com/awesome-open-source

Thanks for sharing~
